src/content/zh-tw/updates/2017/09/picture-in-picture.md

```diff
@@ -0,0 +1,100 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: 讓我們來討論如何使用一個Web API以在桌面創建一個浮動視頻窗口。
+
+{# wf_updated_on: 2017-09-12 #}
+{# wf_published_on: 2017-09-12 #}
+{# wf_tags: news,media #}
+{# wf_featured_image: /web/updates/images/generic/picture-in-picture.png #}
+{# wf_featured_snippet: 讓我們來討論如何使用一個Web API以在桌面創建一個浮動視頻窗口。 #}
+{# wf_blink_components: Blink>Media #}
+
+# 畫中畫 (PiP) {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Translated by
```

Comment:
Per our [style guide](https://developers.google.com/web/resources/translations#credit), please put the translated by at the bottom of the article.